### PR TITLE
Added automodapi flag to conf.py to include inherited members in docs…

### PR DESCRIPTION
**Pull request by cczhu**
... and flags to .rst files to include all objects. Fixed formatting issue with baseband/vdif/frame.py docstring.  Changed `baseband.mark4.header.__all__()` to include `CRC12`, `crc12` to be consistent with `baseband.mark5b.header.__all__()`.

### CODE REVIEW

File: baseband/vdif/frame.py [hunk 1]
```diff
@@ -38,6 +38,8 @@ class VDIFFrame(VLBIFrameBase):
         (e.g., that channel information and whether or not data are complex
         are consistent between header and data).  Default: `True`
 
+    Notes
+    -----
     The Frame can also be read instantiated using class methods:
```
Comment (mhvk) [inline baseband/vdif/frame.py]: delete "read" here

**Comment by mhvk** [comment]
@cczhu - looks good, but could you rebase on master to get rid of the first two, unneeded commits? While you are at it, also fix the one nitpick (not due to you)...
```
git fetch upstream
git rebase -i upstream/master
# remove first two unrelated commits
git push -f
```

**Comment by cczhu** [comment]
Done.

**Comment by mhvk** [comment]
Thanks, merging!

**Pull request merged**